apps/web/src/components/editor/index.tsx

```diff
@@ -204,14 +204,17 @@ export default function TabsView() {
               </Panel>
             </>
           )}
+
+          {isTOCVisible && activeSessionId && (
+            <Panel order={3} defaultSize={30}>
```

Comment:
You need to add `PanelResizeHandle` otherwise it won't resize. Also set `defaultSize` to 25 and minSize to 15. It looks too big. You'll also need to add an `id` prop to the handle otherwise the `order` doesn't work.

Reply:
Add `id` prop to `Panel` or `PanelResizeHandle`. I've added `id` to `Panel` as it is in other panels in the component. Let me know if we want to add to `PanelResizeHandle` as well

Reply:
I don't think adding it to `PanelResizeHandle` is necessary.